src/Dev/FunctionalTest.php

```diff
@@ -188,15 +188,16 @@ public function post($url, $data, $headers = null, $session = null, $body = null
      *
      * @see http://www.simpletest.org/en/form_testing_documentation.html
      *
-     * @param string $formID HTML 'id' attribute of a form (loaded through a previous response)
-     * @param string $button HTML 'name' attribute of the button (NOT the 'id' attribute)
-     * @param array $data Map of GET/POST data.
+     * @param $formID HTML 'id' attribute of a form (loaded through a previous response)
+     * @param $button HTML 'name' attribute of the button (NOT the 'id' attribute)
+     * @param $data Map of GET/POST data.
+     * @param $withSecurityToken Submit with the form's security token if there is one.
```

Comment:
```suggestion
     * @param string $formID HTML 'id' attribute of a form (loaded through a previous response)
     * @param string $button HTML 'name' attribute of the button (NOT the 'id' attribute)
     * @param array $data Map of GET/POST data.
     * @param bool $withSecurityToken Submit with the form's security token if there is one.
```
Need param types here as per https://github.com/silverstripe/silverstripe-framework/pull/10643#discussion_r1097958505

Reply:
DONE